osdp/pipeline/pipeline_stack.py

```diff
@@ -12,9 +14,10 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
 
         # Define the CodePipeline source
         source = pipelines.CodePipelineSource.git_hub(
-            "nulib/osdp-prototype-cdk",  
+            "nulib/osdp-prototype-cdk",
             "main",
             authentication=SecretValue.secrets_manager("osdp/github-token"),
+            trigger=actions.GitHubTrigger.WEBHOOK
```

Comment:
This _should_ trigger everything on pushes to `main`, but I'm unsure what needs to be set up to test correctly

Reply:
We need to actually set up a webhook too

Reply:
You read my mind! Reverting…